ddpui/tests/services/test_dashboard_service.py

```
@@ -436,6 +365,947 @@ def test_filter_data_optional_fields(self):
+            org=sep_org,
+        )
+        d2 = Dashboard.objects.create(
+            title="Other",
+            dashboard_type="native",
+            grid_columns=12,
+            layout_config=[],
+            components={},
+            created_by=sep_orguser,
+            org=sep_org,
+        )
+        success, msg = delete_dashboard_safely(d1.id, sep_orguser)
```

Ruff (0.15.9)

[warning] Unpacked variable `msg` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)